lib/Optimizer/CAPI/CMakeLists.txt

```diff
@@ -15,4 +15,4 @@ add_mlir_public_c_api_library(CUDAQuantumMLIRCAPI
   LINK_LIBS PRIVATE 
   QuakeDialect 
   CCDialect
-)
\ No newline at end of file
+)
```

Comment:
Wonder if this can be enforced by a pre-commit hook?

Reply:
Seems like it could be. 🤷